### DISCUSSION TITLE
After upgrading to 1.7.1 making a selection makes bootstrap dropdown disappear

### DISCUSSION THREAD

**Discussion by acetinick** [orchidjs/tom-select#102] [General]
I have some forms that are embeded inside a bootstrap dropdown and after upgrading to >= 1.7.1 ,  once you make a selection of the tom-select field inside the dropdown area, it will close the dropdown.    Before prior to 1.7.1 it would stay open, allowing users to make further selections or submit the form.  Must be some new event triggered that bootstrap dropdown uses to close. 

Is there a way I can configure it to do the previous behaviour? If I remove items or clear items, it works fine, but only when i select.

**Comment by oyejorge** [comment]
Hmm. I haven't been able to reproduce the behavior and I might need some help figuring out where the problem is. Do you  have the some problem with the example on https://tom-select.js.org? When you say "bootstrap dropdown", are you using bootstrap classes for the dropdown and/or bootstrap javascript for dropdown placement? Do you have a custom option render template?

**Reply by oyejorge** [reply]
> embeded inside a bootstrap dropdown

right, like this? https://jsfiddle.net/4dbt5os2/

**Reply by oyejorge** [reply]
Fixed and tests added with [this commit](https://github.com/orchidjs/tom-select/commit/10149a2937e48e958a8b800d5e9e919594021f55)

**Reply by acetinick** [reply]
Yep thats the scenario! Awesome, looking forward to trying it out in the next release!

**Reply by oyejorge** [reply]
Just pushed out 1.7.3!

**Reply by acetinick** [reply]
perfect works great) thanks @oyejorge !
